quantum/q_full_adder.py

```diff
@@ -9,26 +9,26 @@
 """
 
 import qiskit
-from qiskit import Aer, ClassicalRegister, QuantumCircuit, QuantumRegister, execute
-
+from qiskit import QuantumCircuit, QuantumRegister, ClassicalRegister, execute, Aer
+import math
```

Comment:
Kinda cool how pre-commit fixes all this stuff… trailing whitespace, unsorted imports, single quotes, etc.